### PR TITLE
Initial work on the getting started section

### PR DESCRIPTION
**Pull request by v-bpastiu**
Work here reflects this ticket:
https://github.com/digitalocean/k8s-adoption-journey/issues/2

### CODE REVIEW

File: docs/getting-started/installing-required-tools.md [hunk 1]
```diff
@@ -10,6 +10,28 @@ To complete this section, you will need:
 2. A [DigitalOcean personal access token](https://docs.digitalocean.com/reference/api/create-personal-access-token) for using the `DigitalOcean` API.
 3. Curl package installed on your system.
 
+## Installing Docker Desktop
+
+Depending on your operating system, you can install [docker-desktop](https://docs.docker.com/desktop/) in the following ways:
+
```
Comment (v-ctiutiu) [inline docs/getting-started/installing-required-tools.md:16]: Please rely on CLI commands wherever possible. On Mac, we can rely on Homebrew. On Linux, we can either use generic `curl` commands, or pick the most popular distro - e.g. Ubuntu.

Please keep the validation steps at the end (i.e `docker version`).

File: docs/getting-started/installing-required-tools.md [hunk 2]
```diff
@@ -10,6 +10,28 @@ To complete this section, you will need:
 2. A [DigitalOcean personal access token](https://docs.digitalocean.com/reference/api/create-personal-access-token) for using the `DigitalOcean` API.
```
Comment (v-ctiutiu) [inline docs/getting-started/installing-required-tools.md:10]: The auth and tokens part should be moved to authentication chapter.

File: docs/getting-started/setting-up-a-docr-registry.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+This section will show you how to create a DOCR registry using the `doctl` CLI.
+
+From the command line run the following:
+
+```shell
+doctl registry create <YOUR_REGISTRY_NAME> --subscription-tier basic
+```
+
+**Note:**
```
Comment (v-ctiutiu) [inline docs/getting-started/setting-up-a-docr-registry.md:11]: Please use [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) from Material theme for notes. We should benefit from all features provided by the [Material](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) theme.

File: docs/getting-started/setting-up-a-doks-cluster.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+```shell
+doctl k8s cluster create <YOUR_CLUSTER_NAME> \
+  --auto-upgrade=false \
+  --maintenance-window "saturday=21:00" \
+  --node-pool "name=basicnp;size=s-2vcpu-2gb;count=1;tag=adoption-journey;label=type=basic" \
+  --region nyc1
+```
+
+**Note:**
```
Comment (v-ctiutiu) [inline docs/getting-started/setting-up-a-doks-cluster.md:15]: Please use [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) from Material theme for notes. We should benefit from all features provided by the [Material](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) theme.

File: docs/getting-started/testing-the-setup.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+This section will show you how to quickly install a sample application to test that the setup works.
+You will install the 2048 game application on your newly created Kubernetes cluster.
+
+1. Clone the [sample-apps-repository](https://github.com/digitalocean/kubernetes-sample-apps)
+2. Navigate to the `game-2048-example` and run the steps described in the repository's [README](https://github.com/digitalocean/kubernetes-sample-apps/blob/master/game-2048-example/README.md).
+
+Running the instructions in the sample app repository step by step will validate that everything is working correctly. You should be able to deploy the `game-2048` application in your cluster using a docker image from your DOCR account.
+
+**Note:**
```
Comment (v-ctiutiu) [inline docs/getting-started/testing-the-setup.md:11]: Please use [admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) from Material theme for notes. We should benefit from all features provided by the [Material](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) theme.

**Review by v-ctiutiu — CHANGES_REQUESTED** [review]

File: docs/getting-started/installing-required-tools.md (hunk 1 above)
Comment (v-bpastiu) [reply docs/getting-started/installing-required-tools.md:15]: Fixed in the latest commit. Thanks

File: docs/getting-started/installing-required-tools.md (hunk 2 above)
Comment (v-bpastiu) [reply docs/getting-started/installing-required-tools.md]: Fixed in the latest commit. Thanks

File: docs/getting-started/setting-up-a-doks-cluster.md (hunk 4 above)
Comment (v-bpastiu) [reply docs/getting-started/setting-up-a-doks-cluster.md]: Fixed in the latest commit. Thanks

File: docs/getting-started/testing-the-setup.md (hunk 5 above)
Comment (v-bpastiu) [reply docs/getting-started/testing-the-setup.md]: Fixed in the latest commit. Thanks

**Review by v-ctiutiu — APPROVED** [review]
Looks good to me now. Thanks.

**Pull request merged**